### ISSUE TITLE
Automatically copy kernels with <ORIGINAL> tags

### ISSUE THREAD

**Issue by HariNairJHUAPL** [NASA-PDS/naif-pds4-bundler#24]
When generating the PDS delivery, I need to rename files on our local filesystem to more NAIF-friendly names.  The <ORIGINAL> tag in the <kernel> blocks of the configuration XML file allows me to map the old and new filenames, but I still have to copy the old filename to the new one.  My setup script parses the XML, looks for the <ORIGINAL> tag, and copies the file properly, but it would be nice if the naif-pds4-bundler would do this at the time it generates the archive.

**Comment by marcsit** [comment]
@HariNairJHUAPL you mean that you want to be able to use the new kernel names in the release plan, instead of the old ones correct?

**Comment by HariNairJHUAPL** [comment]
Yes, for example, the original kernel name is LCC220630-FK-V1.tf and I want the name in the archive to be licia_003.tf.  I can specify this in the XML configuration, but I still have to copy the file to its new name myself.  It seems like this would be an easy thing for the NPB script to handle. 

              <ORIGINAL value="licia_003.tf">LCC220630-FK-V1.tf</ORIGINAL>

Thanks!  Hope all is well with your new position.

**Comment by marcsit** [comment]
@HariNairJHUAPL Indeed this is a known issue and it was annoying me at some point as well so I will implement it.

The new position is going very well for the time being :)
